extensions/functions_aggregate_generic.yaml

```diff
@@ -22,3 +22,23 @@ aggregate_functions:
         decomposable: MANY
         intermediate: i64
         return: i64
+  - name: "grouping"
+    description: "Indicates whether the column specified in a GROUP BY clause aggregated or not"
+    impls:
+      - args:
+          - options: [ SILENT, ERROR ]
+            required: false
+          - value: any
+        nullability: DECLARED_OUTPUT
+        return: i64
```

Comment:
```suggestion
        return: boolean
```

Reply:
I chose to use `i64` because calcite is `BIGINT`, see [here](https://github.com/apache/calcite/blob/b9c2099ea92a575084b55a206efc5dd341c0df62/core/src/main/java/org/apache/calcite/sql/fun/SqlGroupingFunction.java#L43).

Reply:
+1 for boolean. Judging by the docs you linked, it only does that because it's variadic and uses one bit for each argument. The way you defined the function here it only takes a single argument, and thus only needs to output a single bit, which is what booleans are meant for. I guess you could also make this function variadic, but without a bigint type, the number of arguments would be artificially limited to 64 to avoid overflow.

Reply:
ok, agree, changed it to `boolean` and kept it with single argument.